internal/jptrace/attributes.go

```diff
@@ -17,20 +14,3 @@ const (
 	// e.g. proto, thrift, json.
 	FormatAttribute = "@jaeger@format"
 )
-
-func PcommonMapToPlainMap(attributes pcommon.Map) map[string]string {
```

Comment:
I would've kept it here. Creating too many top packages at internal just creates clutter, and in this case it's a new package for just a couple of functions.

Reply:
@yurishkuro I had to do this because of circular dependencies. `proto-gen` uses `jptrace.TracesData` and this package needs to use `proto-gen` for converting to the key-value list. Let me know if you prefer a different approach. We could create a top-level `jotel` and create sub-packages `jptrace` and `jpcommon` in there. Or alternatively move the `TracesData` out of `jptrace`.